.watchmanconfig

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Presumably you can get shot of this now?

Reply:
I'm using wml to develop locally - and that's what I'm recommending in the cavy-tester readme, so it's nice not to copy these files.

Reply:
Oh wait sorry, was getting confused about watchman vs. wml. Ignore me!